cmd/tk/export.go

```diff
@@ -77,8 +84,14 @@ func exportCmd() *cli.Command {
 			}
 
 			// validate environment
-			if _, err := tanka.Peek(path, opts.Opts.JsonnetOpts); err != nil {
-				return err
+			if _, err := tanka.LoadEnvironment(path, opts.Opts); err != nil {
```

Comment:
Not using `Peek` has the consequence that `static` environments are fully evaluated – which is costly for discarding that result afterwards.

What's the reasoning behind switching to `LoadEnvironments`? `Peek` should return the exact same error

Reply:
Because of `opts.Name`, as you found out below. Looking for suggestions.

Reply:
If `Peek` would take an alternative `EvalScript`, we could circumvent that performance issue. WDYT?

Reply:
Suggestion:

Make the `tanka.Peek` (not `Loader.Peek`) accept `tanka.Opts` instead of `jsonnet.Opts`, so you can pass your `Name` field. Unsure why it wasn't this way already.

Then create: 
```go
type LoaderOpts struct {
  JsonnetOpts
  Name string
}
```

The inline implementation can use that, the static implementation can happily disregard the field

Reply:
Or if it would take `tanka.Opts`, we could move that in to the loader.

Reply:
Great when we align just like that.